src/providers/dataProvider/index.ts

```diff
@@ -100,11 +101,21 @@ export const getDataProvider = async (): Promise<DataProvider<string>> => {
         )
         return record
       },
-      afterCreate: async (record: CreateResult<Project>) => {
+      afterCreate: async (
+        record: CreateResult<Project>,
+        dataProvider: DataProvider
+      ) => {
+        const { data } = record
+        const { id } = data
         await audit(
           AuditType.CREATE_PROJECT,
           `Project created (${String(record.data.id)})`
         )
+        await dataProvider.update<Project>('projects', {
+          id,
+          previousData: data,
+          data: { created_at: DateTime.now().toFormat('yyyy-MM-dd') }
+        })
         return record
       },
       afterUpdate: async (record: UpdateResult<Project>) => {
```

Comment:
@TahaKhanAbdalli - just to help my react-admin skills....  Is there a reason why you inject the `created_at` data in the `afterCreate`, rather than `beforeCreate`?